packages/cli/src/yargs/logCommandDebugMiddlewares.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Arguments } from "yargs";
+import type { CommonSiteArgs } from "../commands/site/CommonSiteArgs.js";
+import type { SiteDeployArgs } from "../commands/site/deploy/SiteDeployArgs.js";
+import type { GitDescribeAutoVersionConfig } from "../util/config.js";
+
+export async function logSiteCommandDebugMiddleware(
```

Comment:
Hmm, these feel extremely specific to the single command they live in so I think it would be fine and easier to follow to just define them in the same file as the command